stdlib/streams/src/main/java/org/ballerinalang/nativeimpl/Deserialize.java

```diff
@@ -21,25 +21,15 @@
 import org.ballerinalang.jvm.scheduling.Strand;
 import org.ballerinalang.jvm.values.MapValue;
 import org.ballerinalang.jvm.values.MapValueImpl;
-import org.ballerinalang.model.types.TypeKind;
-import org.ballerinalang.natives.annotations.Argument;
-import org.ballerinalang.natives.annotations.BallerinaFunction;
-import org.ballerinalang.natives.annotations.ReturnType;
 
 /**
  * ballerina.model.streams:deserialize(string str).
  *
  * @since 0.990.5
  */
-@BallerinaFunction(
-        orgName = "ballerina", packageName = "streams",
-        functionName = "deserialize",
-        args = {@Argument(name = "serializedString", type = TypeKind.STRING)},
-        returnType = {@ReturnType(type = TypeKind.MAP)},
-        isPublic = true
-)
 public class Deserialize {
 
+    @Deprecated
     public static MapValue deserialize(Strand strand, Object object, String str) {
```

Comment:
If we replace Strand with the Object, class we should be able to remove the  JBallerina runtime dependency as well.